sim/death_knight/blood/TestBlood.results

```diff
@@ -2293,191 +2293,191 @@ dps_results: {
  }
 }
 dps_results: {
- key: "TestBlood-Settings-Orc-p1-Basic-simple-FullBuffs-0.0yards-LongMultiTarget"
+ key: "TestBlood-Settings-Orc-p1-Basic-simple--FullBuffs-0.0yards-LongMultiTarget"
```

Comment:
Search "Don't append name here unless" within sim/core/test_generators.go to fix this.